<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QComboBox" name="DefaultProfile_Test_1_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_1_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_1_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_2_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_2_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_2_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_3_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_3_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_3_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_4_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_4_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_4_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Block Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Queues</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Threads</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>471</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="MeasuringTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="IntervalTime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_DefaultProfile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0; color: black</string>
   </property>
   <property name="text">
    <string>Profile: Default</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_1_Pattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_2_Pattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_3_Pattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DefaultProfile_Test_4_Pattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_PeakPerformanceProfile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0; color: black</string>
   </property>
   <property name="text">
    <string>Profile: Peak Performance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_1_Pattern">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_2_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_1_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_1_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_2_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_2_Pattern">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_1_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="PeakPerformanceProfile_Test_2_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_DemoProfile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0; color: black</string>
   </property>
   <property name="text">
    <string>Profile: Demo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DemoProfile_Test_1_Pattern">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DemoProfile_Test_1_Queues">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DemoProfile_Test_1_Threads">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DemoProfile_Test_1_BlockSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_MeasuringTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0; color: black</string>
   </property>
   <property name="text">
    <string>Measuring time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_IntervalTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f0f0; color: black</string>
   </property>
   <property name="text">
    <string>Interval time</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>DefaultProfile_Test_1_Pattern</tabstop>
  <tabstop>DefaultProfile_Test_1_BlockSize</tabstop>
  <tabstop>DefaultProfile_Test_1_Queues</tabstop>
  <tabstop>DefaultProfile_Test_1_Threads</tabstop>
  <tabstop>DefaultProfile_Test_2_Pattern</tabstop>
  <tabstop>DefaultProfile_Test_2_BlockSize</tabstop>
  <tabstop>DefaultProfile_Test_2_Queues</tabstop>
  <tabstop>DefaultProfile_Test_2_Threads</tabstop>
  <tabstop>DefaultProfile_Test_3_Pattern</tabstop>
  <tabstop>DefaultProfile_Test_3_BlockSize</tabstop>
  <tabstop>DefaultProfile_Test_3_Queues</tabstop>
  <tabstop>DefaultProfile_Test_3_Threads</tabstop>
  <tabstop>DefaultProfile_Test_4_Pattern</tabstop>
  <tabstop>DefaultProfile_Test_4_BlockSize</tabstop>
  <tabstop>DefaultProfile_Test_4_Queues</tabstop>
  <tabstop>DefaultProfile_Test_4_Threads</tabstop>
  <tabstop>PeakPerformanceProfile_Test_1_Pattern</tabstop>
  <tabstop>PeakPerformanceProfile_Test_1_BlockSize</tabstop>
  <tabstop>PeakPerformanceProfile_Test_1_Queues</tabstop>
  <tabstop>PeakPerformanceProfile_Test_1_Threads</tabstop>
  <tabstop>PeakPerformanceProfile_Test_2_Pattern</tabstop>
  <tabstop>PeakPerformanceProfile_Test_2_BlockSize</tabstop>
  <tabstop>PeakPerformanceProfile_Test_2_Queues</tabstop>
  <tabstop>PeakPerformanceProfile_Test_2_Threads</tabstop>
  <tabstop>DemoProfile_Test_1_Pattern</tabstop>
  <tabstop>DemoProfile_Test_1_BlockSize</tabstop>
  <tabstop>DemoProfile_Test_1_Queues</tabstop>
  <tabstop>DemoProfile_Test_1_Threads</tabstop>
  <tabstop>MeasuringTime</tabstop>
  <tabstop>IntervalTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
